wave_viewer.m

```diff
@@ -2191,6 +2201,7 @@ function set_viewer_axlims(hax,t_min,t_max,dat4minmax,yax_fact2use)
 plot_params.yes_gray = 1;
 plot_params.thresh_gray = 0;
 plot_params.max_gray = 1;
+plot_params.show_formants = 1;
```

Comment:
Suggest "yes_show_formants" in keeping with this function's convention for faux booleans